iOS/Plugins/FlipperKitUserDefaultsPlugin/FKUserDefaultsSwizzleUtility.m

```diff
@@ -0,0 +1,59 @@
+//
+//  FKUserDefaultsSwizzleUtility.m
+//  FlipperKit
+//
+//  Created by Marc Terns on 10/6/18.
+//
+
+#import "FKUserDefaultsSwizzleUtility.h"
+#import <objc/runtime.h>
+
+static NSMutableSet *swizzledClasses;
+static NSMutableDictionary *swizzledBlocks;
```

Comment:
Can we get rid off this static variables ?

Reply:
I can build a singleton and declare the properties as strong/copy. Then I can have a private instance method to do the swizzling. The swizzleutility needs to be a singleton since it keeps references of all the blocks to be called once the swizzling happens and we can not lose those references whatsoever(they need to be alive as long as the process is alive). 

I can add also comments to every single line of the FKUserDefaultsSwizzleUtility to better explain what it does.

Reply:
Hey @priteshrnandgaonkar, I made the changes as requested in 12ec6096af378f4e09d92c18381f23215f43ffdf